pkgpanda/__init__.py

```diff
@@ -356,7 +356,7 @@ def load(self, id: str):
         filename = os.path.join(path, "pkginfo.json")
         try:
             pkginfo = load_json(filename)
-        except FileNotFoundError as ex:
+        except OSError as ex:
```

Comment:
The `FileNotFoundError` captures just the file not found, while `OSError` captures the file with bad permissions to read.  I don't know if if the latter with occur in practice.  The status quo might be better here IMO. Also, I see all placed were we use load_json in code base looking for `FileNotFoundError` only.